### PR TITLE
Add Mississauga recommendations

### PR DESCRIPTION
**Pull request by dbriand**
This Pull Request closes issue #48.
### Adding restaurants and activities
- Does anyone have a favorite food?
- Are you wanting outdoor activities or indoor activities

### CODE REVIEW

File: dbriand/dbriand-mississauga.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Mississauga Ontario  :heart:
+
+#### Restaurants
+
+#### Things to do
```
Comment (Apple-K) [inline dbriand/dbriand-mississauga.md:8]: @dbriand is Niagara Falls far from you?

File: dbriand/dbriand-mississauga.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Mississauga Ontario  :heart:
```
Comment (ghost) [inline dbriand/dbriand-mississauga.md:1]: what kind of food is typical from Missisauga?

File: dbriand/dbriand-mississauga.md (hunk 1 above)
Comment (dbriand) [inline dbriand/dbriand-mississauga.md:8]: Very close <100km

File: dbriand/dbriand-mississauga.md (hunk 2 above)
Comment (dbriand) [inline dbriand/dbriand-mississauga.md:1]: Very multi-cultural ... just about anything .. lots of Indian food.

**Pull request merged**